packages/ui/src/partials/textbox.tsx

```diff
@@ -9,7 +9,9 @@ const textVariants = cva(
     variants: {
       variant: {
         default: "text-base text-black",
-        description: "text-sm text-gray-400",
+        description: "text-sm text-gray-600",
+        link: "inline text-stone-500",
```

Comment:
```suggestion
        link: "inline text-stone-500",
```
is variant type link needed for textbox?